.bit/responses/4.2-Week 4 Step 2.md

```diff
@@ -1,22 +1,68 @@
 ---
-files: 
-stepType: 
-scripts: 
+files: n/a
+stepType: checks
+scripts: 4.2.spec.js
 week: 4
 step: 2
 name: Week 4 Step 2
 ---
+Week 4 Step 2 ⬤⬤◯◯◯◯ | 🕐 Estimated completion: 20-25 minutes
 
-## Week 4 Step 2
+## Error Handling ~ Sir this is a Wendy's
+Ideally, users will fill the web form with the correct information. However, people make mistakes. In this step, we will be implementing some form validation to make sure that a user has inputted text value before submitting their password. 
 
-### Submitting an MVP
+### ✅  Task:
+- [ ]  Check you are on the `week4` branch 
+- [ ] Change your HTML form to accept `multipart/form-data` in `index.html`
+- [ ] Add a file input to `index.html` that accepts `image/x-png,image/jpeg` 
```

Comment:
The button must also be modified, since now you have to submit a form.
The type used to be `button` but now it must be `submit`

```html
<input type="submit" value="Submit Your Picture 📷"></input>
```